docs/generator.go

```diff
@@ -21,43 +21,53 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
+	// detect if file exists
+	_, err = os.Stat(tmpl)
+	// create file if not exists
+	if os.IsNotExist(err) {
```

Comment:
Maybe this could be inlined this way ?

```go
if _, err := os.Stat(tmpl); os.IsNotExist(err) {
    ...
}
```

More of a proposal, not an obligation. This could make it harder to read. Thoughts ?

Reply:
One is shorter and the other is more readable. I don't favor one or the other. Maybe a tiny little bit to the shorter inlined way.

Reply:
Then we go for it :)